docs/tutorial.rst

```diff
@@ -0,0 +1,912 @@
+  >>> # extends axis limits 5% beyond given limits
+  >>> plt.autoscale() # doctest: +SKIP
+  >>> plt.show() # doctest: +SKIP
+
+  >>> print(sm.Ncomponents)
+  4597
+  >>> print(sm.Nfreqs)
+  1
+  >>> print(sm.component_type)
+  point
+  >>> print(sm.spectral_type)
+  spectral_index
+  >>> # correction done since catalog reference frequencies had wrong power
+  >>> sm.reference_frequency = sm.reference_frequency*10**6
+  >>> print(np.unique(sm.reference_frequency))
+  [1.82435013e+08 2.15675003e+08] Hz
+  >>> print(np.unique(sm.spectral_index))
+  [-0.8]
+  >>> print(np.unique(sm.extended_model_group))
+  ['32768' '32769' '32770']
+  >>> # beam amplitude for stokes parameter index = 0 (stokes I or unpolarized), Nfreqs index = 0 (first and only frequency
+  >>> # since Nfreqs = 1 for spectral index type), Ncomponents index = : (all components)
+  >>> print(np.unique(sm.beam_amp[0,0,:]))
+  [0.12874769 0.56623143 0.59106636]
+  >>> # coherency (2 x 2 matrix of electric field correlation) for polarization1 index = 0 (North),
+  >>> # polarization2 index = 0 (North), Nfreqs index = 0 (first and only frequency since Nfreqs = 1 for
+  >>> # spectral index type), Ncomponents index = 0 (first component)
+  >>> print(sm.coherency_radec[0,0,0,0])
+  (8.400908470153809+0j) Jy
+
+  >>> # dividing by 10^6 since frequency typically plotted in units of MHz
+  >>> plt.scatter(sm.reference_frequency/10**6, sm.spectral_index) # doctest: +SKIP
+  >>> plt.xlabel("Reference Frequency (MHz)") # doctest: +SKIP
+  >>> plt.ylabel("Spectral Index") # doctest: +SKIP
+  >>> plt.show() # doctest: +SKIP
+
+  >>> index_32768 = []
+  >>> for j in range(len(sm.extended_model_group)):
+  ...     if sm.extended_model_group[j] == "32768":
+  ...         # indices for extended model group 32768
+  ...         index_32768.append(j)
+
+  >>> # confirming that there is one reference frequency for this extended model group
+  >>> print(np.unique(sm.reference_frequency[index_32768]))
+  [2.15675003e+08] Hz
+
+  >>> # plots of fluxes are sensible at one frequency since fluxes can change with frequency, plots below provide fluxes
+  >>> # when frequency = reference frequency (more on this in at_frequencies section)
+
+  >>> # log taken since these fluxes have different orders of magnitude
+  >>> plt.hist(np.log(sm.stokes.value[0,0,index_32768]), bins=20) # doctest: +SKIP
+  >>> plt.xlabel("log(Flux (Jy))") # doctest: +SKIP
+  >>> plt.ylabel("Counts") # doctest: +SKIP
+  >>> plt.show() # doctest: +SKIP
+
+  >>> plt.scatter(x=sm.ra[index_32768],y=sm.dec[index_32768],c=sm.stokes[0,0,index_32768],cmap="plasma") # doctest: +SKIP
+  >>> cbar=plt.colorbar(label="Flux (Jy)", orientation="vertical",shrink=.75) # doctest: +SKIP
+  >>> plt.xlim(max(sm.ra.value[index_32768]), min(sm.ra.value[index_32768])) # doctest: +SKIP
+  >>> plt.autoscale() # doctest: +SKIP
+  >>> plt.xlabel("RA (deg)") # doctest: +SKIP
+  >>> plt.ylabel("DEC (deg)") # doctest: +SKIP
+  >>> plt.show() # doctest: +SKIP
```

Comment:
These plots are great, I think it'd be nice to include the files from them in the tutorial so users can see what theirs should look like. There's an example of how to include images in rst files here: https://github.com/RadioAstronomySoftwareGroup/pyuvsim/blob/main/docs/comparison.rst

(click the "raw" button to see how it's actually encoded in the file vs the rendered file).